File: Python/Exercicio21.py
"""
Faça um programa que leia um nome de usuário e a sua senha e não
aceite a senha igual ao nome de usuário, mostrando uma mensagem
de erro e voltando a pedir as informações.
"""

usuario = input("Informe o nome de usuario: ")
senha = input("Informe sua senha: ")

while usuario == senha:
    print("Ususario e senha igual!")
    usuario = input("Informe o nome de usuario: ")
    senha = input("Informe sua senha: ")

print("Usuario e senha cadastrados com sucesso!!!")

File: C/Exercicio21.c
/*
Faca um programa que leia um nome de usuario e a sua senha e nao
aceite a senha igual ao nome de usuario, mostrando uma mensagem
de erro e voltando a pedir as informacoes.
*/

#include<stdio.h>
#include<string.h>

int main(){
	
	char nome[50], senha[50];
	
	printf("Informe seu nome:");
	gets(nome);
	
	printf("Informe sua senha:");
	gets(senha);
	
	if(!strcmp(nome, senha)){
		
		printf("Nome e senha nao podem ser iguais!!!\n");
		
		printf("Informe seu nome:");
		gets(nome);
	
		printf("Informe sua senha:");
		gets(senha);
	}else{
		printf("Usuario e senha cadastrados com sucesso!!!");
	}
	
}
